src/sentry/static/sentry/app/data/platforms.tsx

```diff
@@ -20,4 +21,4 @@ export default [].concat(
   ...[...platforms, otherPlatform].map(platform =>
     platform.integrations.map(i => ({...i, language: platform.id}))
   )
-);
+) as PlatformIntegration[];
```

Comment:
heh. I think I tried to do this one even fancier and had a hard time getting this concat to be happy. This is probably fine though

Reply:
I thought of typing some of the intermediate steps but it didn't really change the end result, and I figured this was better than any.